src/Management/test/Endpoint.Test/ContentNegotiationTests.cs

```diff
@@ -0,0 +1,234 @@
+
+        await using WebApplication host = builder.Build();
+        await host.StartAsync();
+
+        var requestMessage = new HttpRequestMessage
+        {
+            Method = HttpMethod.Get,
+            RequestUri = new Uri("http://localhost/actuator/loggers"),
+            Headers =
+            {
+                Accept =
+                {
+                    MediaTypeWithQualityHeaderValue.Parse("text/html; q=0.8"),
+                    MediaTypeWithQualityHeaderValue.Parse("application/json; q=0.1")
+                }
+            }
+        };
+
+        using HttpClient client = host.GetTestClient();
+        HttpResponseMessage response = await client.SendAsync(requestMessage);
+
+        response.StatusCode.Should().Be(HttpStatusCode.OK);
+    }
+
+    [Fact]
+    public async Task Can_use_wildcard_accept_header()
+    {
+        WebApplicationBuilder builder = TestWebApplicationBuilderFactory.Create();
+        builder.Configuration.AddInMemoryCollection(AppSettings);
+        builder.Services.AddLoggersActuator();
+
+        await using WebApplication host = builder.Build();
+        await host.StartAsync();
+
+        var requestMessage = new HttpRequestMessage
+        {
+            Method = HttpMethod.Get,
+            RequestUri = new Uri("http://localhost/actuator/loggers"),
+            Headers =
+            {
+                Accept =
+                {
+                    MediaTypeWithQualityHeaderValue.Parse("text/html"),
+                    MediaTypeWithQualityHeaderValue.Parse("*/*; q=0.8")
+                }
+            }
+        };
+
+        using HttpClient client = host.GetTestClient();
+        HttpResponseMessage response = await client.SendAsync(requestMessage);
+
+        response.StatusCode.Should().Be(HttpStatusCode.OK);
+    }
+
+    // AcceptHeader values captured from Spring Boot Admin 3.4.1
+    [InlineData("", $"{ActuatorV3},{SpringBootStandardAccept}", ActuatorV3)]
+    [InlineData("beans", SpringBootStandardAccept, ActuatorV3)]
+    [InlineData("env", SpringBootStandardAccept, ActuatorV3)]
+    [InlineData("health", $"{ActuatorV3},{SpringBootStandardAccept}", ActuatorV3)]
+    [InlineData("heapdump",
+        "text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.7",
+        "application/octet-stream")]
+    [InlineData("httpexchanges", SpringBootStandardAccept, ActuatorV3)]
+    [InlineData("info", $"{ActuatorV3},{SpringBootStandardAccept}", ActuatorV3)]
+    [InlineData("loggers", SpringBootStandardAccept, ActuatorV3)]
+    [InlineData("mappings", SpringBootStandardAccept, ActuatorV2)]
+    [InlineData("metrics", SpringBootStandardAccept, ActuatorV3)]
+    [InlineData("threaddump", SpringBootStandardAccept, ActuatorV3)]
+    [Theory]
+    public async Task Responses_for_SpringBootAdmin_match_expectations(string endpoint, string acceptHeader, string expectedContentType)
```

Comment:
Consider rename of `expectedContentType` to `responseContentType` to clarify. I imagine the tests will also get a `requestContentType` parameter for POSTs (probably easier to extract into separate `Theory`, where the combination of incoming `Content-Type` and `Accept` headers is tested).